app/victoria-metrics/main_test.go

```diff
@@ -241,7 +244,14 @@ func testRead(t *testing.T) {
 				test := x
 				t.Run(test.Name, func(t *testing.T) {
 					t.Parallel()
-					rowContains(t, httpRead(t, testReadHTTPPath, test.Query), test.Result, test.Issue)
+					for _, q := range test.Query {
+						switch true {
+						case strings.HasPrefix(q, "/api/v1/export"):
```

Comment:
I'd prefer to avoid swtich-case or add default case because wrong/unsupported cases would be skipped silently instead of notifying about errors